akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/javadsl/ClusterSharding.scala

```diff
@@ -197,7 +197,12 @@ abstract class ClusterSharding {
    */
   def entityRefFor[A](typeKey: EntityTypeKey[A], entityId: String): EntityRef[A]
 
-  /** The default ShardAllocationStrategy currently is [[LeastShardAllocationStrategy]] however could be changed in the future. */
+  /**
+   * Typed Actor for querying Cluster Sharding state
```

Comment:
I don't think we have to say Typed in such documentation. Everything in this module is typed.